buyer-api/src/utils/routes.js

```diff
@@ -28,4 +28,32 @@ const validateAddress = paramName => async (req, res, next) => {
   return undefined;
 };
 
-export { errorHandler, asyncError, validateAddress };
+/**
+ * Middleware to check if an notarization result is valid or not.
+ * @public
+ */
+const validateFields = () => (req, res, next) => {
+  const notarizationResult = req.body;
+
+  if (!notarizationResult.orderId) {
+    res.status(422).json({
```

Comment:
We are using `express-boom` to standarize error responses, so I would response with smth like:
`res.boom.badData('Validation error', { errors: ['Field "orderId" is missing or malformed'] });`